package.json

```diff
@@ -1,39 +1,21 @@
 {
-  "name": "@aragon/staking",
-  "version": "0.2.9",
-  "description": "",
-  "author": "Aragon Association <legal@aragon.org>",
+  "name": "@aragon/monorepo-staking",
+  "version": "1.0.0",
+  "private": true,
+  "author": "Aragon Assocation <legal@aragon.org>",
   "license": "(GPL-3.0-or-later OR MIT)",
-  "repository": "https://github.com/aragon/staking",
-  "files": [
-    "/abi",
-    "/build",
-    "/contracts",
-    "/interfaces",
-    "/test"
-  ],
+  "devDependencies": {
+    "lerna": "^3.14.1"
```

Comment:
Depending on how you feel, we may not even need lerna in this monorepo.

Reply:
I'm actually using it to link the subgraph to the contracts, see the subgraph PR